libs/payments/ui/src/lib/server/terms-and-privacy.tsx

```diff
@@ -28,28 +28,33 @@ function GenericTerms({
   l10n,
 }: GenericTermsProps) {
   return (
-    <div className="legal-blurb" role="group" aria-labelledby={titleId}>
-      <p className="legal-heading" id={titleId}>
+    <section
+      className="clear-both mt-5 text-xs leading-5 text-center"
+      role="group"
+      aria-labelledby={titleId}
+    >
+      <h4 className="m-0 font-semibold text-grey-500" id={titleId}>
         {l10n.getMessage(titleLocalizationId)?.value?.toString() || title}
-      </p>
+      </h4>
 
-      <p>
+      <ul className="inline m-0 text-grey-400">
```

Comment:
`inline` doesn't seem to do anything in `ul`, perhaps `<ul className="flex justify-center m-0 text-grey-400">` and then `inline` can be removed from the `li` below as well.

Reply:
Keeping the `inline` reduces 1 place we use flex, and inline seems to have the same net effect in the end.